tests/applications/python/fastcompile.sh

```diff
@@ -0,0 +1,72 @@
+echo "Starting...";
+
+mkdir -p /home/lind/lind_project/tests/applications/python/build/lib
+
+modulesarray=('_collectionsmodule' 'operator' 'itertoolsmodule' '_struct' 'mathmodule' 'binascii' 'timemodule' 'cStringIO' '_randommodule' 'arraymodule' 'socketmodule' '_functoolsmodule' 'cPickle' '_struct' 'selectmodule' 'arraymodule' 'unicodedata')
+libsarray=('_collections' 'operator' 'itertools' '_struct' 'math' 'binascii' 'time' 'cStringIO' '_random' 'array' '_socket' '_functools' 'cPickle' '_struct' 'select' 'array' 'unicodedata')
+
+echo Compiling...
+length=${#libsarray[@]}
+for (( i = 0; i < length; i++ ));
+do
+        x86_64-nacl-gcc -DPY_FORMAT_LONG_LONG=ll -std=c99 -fPIC -fno-strict-aliasing -march=x86-64 -mtune=generic -O2 -pipe -DNDEBUG -I. -IInclude -I./Include -c Modules/${modulesarray[i]}.c -o build/${modulesarray[i]}.o
+done
+
+echo Building libraries...
+for (( i = 0; i < length; i++ ));
+do
+        x86_64-nacl-gcc -pthread -shared -Wl,-O1 -Wl,-Bsymbolic-functions -std=c99 build/${modulesarray[i]}.o -L. -lpython2.7 -o build/lib/${libsarray[i]}.so
+done
+
+echo Copying files...
+for (( i = 0; i < length; i++ ));
+do
+        lindfs cp /home/lind/lind_project/tests/applications/python/build/lib/${libsarray[i]}.so /usr/local/lib/python2.7/lib-dynload/${libsarray[i]}.so
+done
```

Comment:
Also for these just iterate over the elements rather than over the indices, e.g. `for var in modulesarray`

Reply:
Makes sense, will add this on the third PR to finalize both flask_scripts and psycopg2_scripts. Btw we can add it to only copying files and compiling .o files, as building library takes the module name and outputs the library's actual name. (like socketmodule.c->socketmodule.o->_socket.so